src/PointwiseFunctions/GeneralRelativity/WeylElectric.hpp

```diff
@@ -41,6 +41,29 @@ void weyl_electric(
     const tnsr::II<DataType, SpatialDim, Frame>&
         inverse_spatial_metric) noexcept;
 
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes a scalar version of the electric part of the Weyl tensor
+ * in vaccuum.
+ *
+ * \details Computes a scalar version of the electric part of the Weyl tensor
+ * in vaccuum \f$EE\f$ as \f$ EE = E^k_{j} * E^j_{k} \f$ where \f$ E_{ij}\f$ is
+ * the electric part of the Weyl tensor.
+ * \note The electric part of the Weyl tensor needs additional terms for
```

Comment:
`\note The electric part of the Weyl tensor in vacuum is available via `gr::weyl_electric()`. The electric part of the Weyl tensor needs additional terms for matter.`